docs/reference/glossary.rst

```diff
@@ -0,0 +1,16 @@
+.. _glossary:
+
+Glossary
+========
+
+.. glossary::
+
+    configuration
+        A "configuration" is roughly equivalent to a
+        :abbr:`SKU (Stock Keeping Unit)`. In most cases, the Ubuntu Certified
```

Comment:
I didn't find any public page explaining these two terms in our context. I think we need to clarify the hierarchical relationship between them, e.g

```
A configuration represents a specific variant or model of a :term:`platform`
```